src/format/to_dom.js

```diff
@@ -109,7 +109,7 @@ class DOMSerializer {
   }
 }
 
-// :: (Node, ?Object) → DOMFragment
+// :: (Node, ?Object) → DocumentFragment
```

Comment:
I saw that for `Node` and `Event`, will handle `DOMFragment` the same way. I suppose it makes sense since it's pretty easy to confuse the different layers on which stuff can happen in PM.